# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/sloccount/tail.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Naur sloccount-2.26.orig/compute_sloc_lang sloccount-2.26/compute_sloc_lang
--- sloccount-2.26.orig/compute_sloc_lang	2004-08-01 04:02:54.000000000 +0100
+++ sloccount-2.26/compute_sloc_lang	2005-04-13 15:55:22.000000000 +0100
@@ -53,7 +53,7 @@
      *) ${language}_count -f ${language}_list.dat > ${language}_outfile.dat
         ;;
    esac
-   tail -1 < ${language}_outfile.dat
+   tail -n 1 < ${language}_outfile.dat
 
   else
    rm -f ${language}_outfile.dat
